.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,57 @@
+name: Bug Report
+description: File a bug report
+labels: [ "🐞 bug", "triage" ]
+body:
+    - type: markdown
+      attributes:
+          value: |
+              Thanks for taking the time to fill out this bug report!
+    - type: textarea
+      attributes:
+        label: Current Behavior
+        description: A concise description of what you're experiencing.
+      validations:
+        required: true
+    - type: textarea
+      attributes:
+        label: Expected Behavior
+        description: A concise description of what you expected to happen.
+      validations:
+        required: true
+    - type: textarea
+      attributes:
+        label: Steps To Reproduce
+        description: Steps to reproduce the behavior.
+        placeholder: |
+          1. In this environment...
+          2. With this config...
+          3. Run '...' (include code example if possible)
+          4. See error...
+        validations:
+          required: false
+    - type: dropdown
+      id: version
+      attributes:
+        label: Version
+        description: What version of RDF4J are you using?
+        options:
+        - 3.7.1
```

Comment:
I think it will be easier on us if we just have a text field instead of a drop down. That way we don't need to keep this updated when we make a new release.

Reply:
Excellent point. I was thinking about adding an option "latest" or something but that doesn't age well of course. A text field is much simpler.

Reply:
Done.